app/assets/stylesheets/components/stories.scss

```diff
@@ -46,6 +46,10 @@
     right: 0;
     bottom: 0;
     left: 0;
+
+    // Prevent long title content skewing layout
+    overflow-wrap: break-word;
```

Comment:
I did entertain adding the `crayons-story__title` class to the hidden link instead, but in the end I didn't because:

- I feel like it opens the door to more unexpected CSS issues (styling a visually hidden and a visually prominent link with the same selectors)
- I want to get rid of these hidden links (for accessibility reasons), and this keeps everything all together and easy to rip out later 😄

Reply:
Does the same comment regarding Safari apply here as in #16771?  If so, it might be helpful because we have `overflow-wrap: break-word;` followed by another `overflow-wrap` directive.

Reply:
Yes, excellent point. I've added the comment here too.